docs/usage/project-namespace-access.md

```diff
@@ -0,0 +1,51 @@
+# Project Namespace Access
```

Comment:
Please add this document also to the index (`README.md`).

Reply:
Typically, we use snake case for the file names (I see there are some counter-examples, but let's try to be "consistent" at least for new things).